pkg/integrations/mssql/common/collector_config.yaml

```diff
@@ -0,0 +1,189 @@
+collector_name: mssql_standard
+
+metrics:
+  - metric_name: mssql_local_time_seconds
+    type: gauge
+    help: 'Local time in seconds since epoch (Unix time).'
+    values: [unix_time]
+    query: |
+      SELECT DATEDIFF(second, '19700101', GETUTCDATE()) AS unix_time
+  - metric_name: mssql_connections
+    type: gauge
+    help: 'Number of active connections.'
+    key_labels:
+      - db
+    values: [count]
+    query: |
+      SELECT DB_NAME(sp.dbid) AS db, COUNT(sp.spid) AS count
+      FROM sys.sysprocesses sp
+      GROUP BY DB_NAME(sp.dbid)
+  #
+  # Collected from sys.dm_os_performance_counters
+  #
+  - metric_name: mssql_deadlocks
```

Comment:
It's usually recomended to add `_total` suffix to counters. See here: https://prometheus.io/docs/practices/naming/#metric-names

Reply:
Add `_total` suffix to all counters.